base/shell/cmd/lang/en-US.rc

```diff
@@ -121,7 +121,7 @@ ERASE [/N /P /T /Q /S /W /Y /Z /A[[:]attributes]] file ...\n\n\
   /S    Delete file from all sub directory\n\
   /A    Select files to be deleted based on attributes.\n\
         attributes\n\
-        R     Read Only files\n\
+        R     Read-only files\n\
```

Comment:
Since you fixed this "Read only <something>", you may double-check the following list:
https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=read+only+&sr=1
It seems there are a bit more files that may need a similar fix.

Reply:
Thanks @HBelusca for the review.
Depending on context, it may or may not need a hyphen:
- ‘Read-only file selected’ (Adjective - computing).
- ‘Read only the first line’ (Verb + Adverb)

I don't have time to look at this currently. Perhaps an \#EasyHack to note somewhere?